packs/classfeatures/arcane-school.json

```diff
@@ -34,6 +53,11 @@
                 },
                 "flag": "arcaneSchool",
                 "key": "ChoiceSet",
+                "predicate": [
+                    {
+                        "not": "class:wizard"
+                    }
+                ],
```

Comment:
We can (and should) do this cleaner in a single ChoiceSet instead of having two with the same flag.